changelog.md

```diff
@@ -14,6 +14,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [Fix HTTP caching failure when using the last-modified header as the validator](https://github.com/ballerina-platform/ballerina-standard-library/issues/2402)
 - [Fix HTTP caching failure after the initial Max Age expiry](https://github.com/ballerina-platform/ballerina-standard-library/issues/2435)
 
+### Changed
+- [Only allow default path in interceptors engaged at listener level](https://github.com/ballerina-platform/ballerina-standard-library/issues/2452)
```

Comment:
Add `Rename RequestContext add function to set` also under changes

Reply:
Fixed in [`855043d`](https://github.com/ballerina-platform/module-ballerina-http/pull/766/commits/855043d9edf192b3b39872d7dd9dbb67589dd040)